frontend/src/PopUp.js

```diff
@@ -1,16 +1,13 @@
 import React from 'react';
 import Modal from "react-bootstrap/Modal";
-import { Link } from 'react-router-dom';
 import Button from './Button';
 
-function PopUp({ body, sentSuccessfully, setSent, path }) {
+function PopUp({ body, handleClick }) {
   return (
     <Modal show={true}>
```

Comment:
ennek a show-nak itt nem sok értelmét látom... enélkül is látszódnia kéne nem? 🤔

Reply:
Enélkül nem látszódik.

Reply:
igen az biztos, és a popup láthatóságát akkor nem ezzel szabályozzátok